Mage/src/main/java/mage/game/permanent/token/DogToken2.java

```diff
@@ -0,0 +1,31 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+/**
+ *
+ * @author htrajan
+ */
+public final class DogToken2 extends TokenImpl {
```

Comment:
There are only one dog token in the xmage, so no needs to add 2 at the end. Try to rename WhiteDogToken like green cat before (see comment above) or DogToken.

Reply:
That's ok, I miss another DogToken. You can keep same name or rename to WhiteDogToken.

Reply:
renamed to `WhiteDogToken`

Reply:
Please, rename HoundToken to GreenDogToken in `card-pictures-tok.txt`. It was broken after rename from Hound to Dog in on of the last commit (74b6626020f3baed2b9880ca5aa372e921118aa3).

Reply:
sorry, that was my bad

Reply:
fixed